### DISCUSSION TITLE
유저 드롭다운에서 초대 알림 보여주기 (X)

### DISCUSSION THREAD

**Discussion by un0211** [Part3-Team15/taskify#170] [Ideas]
아까 음성으로 말씀드렸던 건데, 지윤님은 못듣기도 했고 여기서 이야기 나누려고 글로 정리합니다!

저희가 기존에 내 대시보드 / 계정관리 접근할 방법이 없어서 드롭다운에 넣었는데,
이제 사이드바에 추가된 만큼 드롭다운에 굳이 넣지 않아도 될 것 같습니다.
<img width="624" alt="image" src="https://github.com/Part3-Team15/taskify/assets/24778465/13132c91-28be-446e-b8f6-7f8a25268b26">

그리고 가장 작은 화면 기준으로 초대 알림을 보여줄 요소를 넣을 만한 공간이 없습니다.
<img width="275" alt="image" src="https://github.com/Part3-Team15/taskify/assets/24778465/6901911f-a713-47a1-9fad-6eac7915dea7">

그래서 드롭다운에 로그아웃만 아래에 남겨둔 채 초대 알림을 볼 수 있게 하면 어떨까 합니다!
초대 내역은 내 대시보드에서 볼 수 있으니 최근 업데이트된 초대만 보여줘도 될 것 같습니다.

**Comment by wayandway** [comment]
기록해주셔서 감사합니다 혜원님! 사이드로 빠졌으니 드롭다운에 알림 넣는 것 좋네요!! 코드잇 프로필 드롭다운도 유사하게 돼있었던 것 같아요

**Comment by wayandway** [comment]
<img width="563" alt="스크린샷 2024-07-02 오후 3 37 06" src="https://github.com/Part3-Team15/taskify/assets/64190056/2e43931e-f787-4e04-9ba1-a19c2b4cdf5a">
앗 참고로 헤더 "관리", "초대하기" 버튼이 아이폰 일부 모델에서 깨지더라구요..! 이것도 모바일에서는 아이콘모양으로 바꾸는 등 수정하면 좋을 것 같아요!

**Reply by un0211** [reply]
아이콘으로 바꿔서 크기를 줄이면, 알림용 드롭다운도 하나 넣을 수 있을 것 같네요!!
<img width="514" alt="image" src="https://github.com/Part3-Team15/taskify/assets/24778465/b12e76b2-1bc7-46e5-88f2-33290ebd9295">

모바일 깨지는 건 여기서 해결했어요!
- #169

패딩을 줄였는데 아이콘으로 바꾸는 것 추가할게요!!
